apps/i18n/applab/en_us.json

```diff
@@ -18,8 +18,133 @@
   "designToolboxHeader": "Design Toolbox",
   "designWorkspaceDescription": "Click on an element or choose it from the dropdown below to edit its properties. Create code in Events.",
   "designWorkspaceHeader": "Workspace",
+  "designWorkspace_copyToScreenButton": "Copy to screen",
+  "designWorkspace_deleteButton": "Delete",
```

Comment:
This string already exists in the common json file; https://github.com/code-dot-org/code-dot-org/blob/669dfc8c43cb3beb02305bee63d6b62d8efc208b/apps/i18n/common/en_us.json#L705